### ISSUE TITLE
DeepDAO Feature Requests and Bug Reports

### ISSUE THREAD

**Issue by namesty** [aragon/connect#309]
DeepDAO reported the following bugs:

- Sometimes, when fetching votes, and their descriptions the process is killed. The more proposals or votes, the more likely it is to happen. Seems to be a server load issue, specially because it comes accompanied by a quota warning coming from the ethereum node.
- Currently, vote descriptions are not working at all.

I suggest (this is also in the feature request document) that Aragon Connect's configuration allows the user to configure his own ethereum node to get around this server load issue. In parallel we should optimize our algorithms.

DeepDAO feature requests with highest priority are:

- Return the address of the vote's proposer from the API; currently it's returning the address of the app that created the proposal.
- Return the vote descriptions in the returned metadata JSON.
- Return all casts associated with a proposal in a single API call.

Lower priority requests, nice-to-haves and others are described in the full document: https://docs.google.com/document/d/1T6aOlO5qEFkoS9bT3yOChsZ_ciy9HQCYdOj2iQ9XGdU/edit#heading=h.fbyakov351h5

**Comment by jessebmiller** [comment]
Thanks @nestorbe I'll get these written out in Linear

**Comment by githubdoramon** [comment]
Can you guys please add here what is the function or endpoint that is failing?
